application/lib/model/User.dart

```diff
@@ -0,0 +1,28 @@
+class User {
+  User.fromJson(Map<String, dynamic> json)
+      : firstName = json['firstName'],
+        lastName = json['lastName'],
+        username = json['username'],
+        token = json['token'];
+
+  final String username;
+  final String firstName;
+  final String lastName;
+  //final String email;
```

Comment:
Hvis den ikke skal bruges, så slet den, ellers brug den til noget. Vi har ikke brug for "død" kode :) 
```suggestion
```

Reply:
Der var snak om dette skulle være en del af databasen eller ej, og det er ikke besluttet endnu

Reply:
Jeg har fjernet den, så kan vi tilføje den senere